### DISCUSSION TITLE
Can we get a custom css override setting

### DISCUSSION THREAD

**Discussion by northkode** [fabiocaccamo/django-admin-interface#191] [Ideas]
The idea here would be to allow us to add some custom css that this plugin adds to the bottom of the style tag that would easily allow CSS overrides to the default theme

**Comment by fabiocaccamo** [comment]
Hi @northkode, 
FYI, this has already been (partially) discussed in #63, #89, #93 and #138 - where you can read some interesting considerations/ideas about the possible approaches.

This would be surely a useful option to have, but considering also the drawbacks, I'm not totally in favour:
- Inline style
- Potentially broken pages style
- Less collaboration by users

So ...

- If you have some generic style improvements, you should:
  - Suggest some improvements opening a new issue.
  - Submit a PR.

- If you have some project-specific style customizations, you could:
  - If you need some global styles, just extend the admin template and include your global CSS file.
  - If you need some style for a specific model, you can define assets for the related ModelAdmin class: https://docs.djangoproject.com/en/3.2/ref/contrib/admin/#modeladmin-asset-definitions
  
**That said, I'm always open to change my mind if there are some good reasons/explanations.**
